src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -140,6 +140,44 @@ namespace winrt::TerminalApp::implementation
         }
         CATCH_LOG();
 
+        if (_settings.GlobalSettings().UseAcrylicInTabRow())
```

Comment:
Out of curiosity, will this setting hot-reload? (i.e. if the user changes the `settings.json` file while the Terminal is open, will the new value apply?)

I'd suspect that moving this to `TerminalPage::_RefreshUIForSettingsReload` might take care of that though

Reply:
This won't hot-reload.
I'm guessing moving it to `TerminalPage::_RefreshUIForSettingsReload` would make hot-reloading work for enabling it but would require additional changes for disabling it.

Reply:
> but would require additional changes for disabling it.

that's a really good point. I'll say :shipit: for now and we can re-visit hot-reloading in a follow-up.